vllm/v1/sample/logits_processor/__init__.py

```diff
@@ -36,6 +40,78 @@
 ]
 
 
+class CallableLogitsProcessor(LogitsProcessor):
+
+    def __init__(self, vllm_config: "VllmConfig", device: torch.device,
+                 is_pin_memory: bool, callable: LogitsProcessorV0,
+                 is_argmax_invariant: bool):
+        self.callable = callable
+        self._is_argmax_invariant = is_argmax_invariant
+
+        # index -> (min_toks, output_token_ids, stop_token_ids)
+        self.device = device
+        self.pin_memory = is_pin_memory
+        self.min_toks: dict[int, tuple[int, Sequence[int], set[int]]] = {}
+
+        # (req_idx_tensor,eos_tok_id_tensor)
+        self.logits_slice: tuple[torch.Tensor,
+                                 torch.Tensor] = (self._device_tensor(
+                                     [], torch.int32),
+                                                  self._device_tensor(
+                                                      [], torch.int32))
+
+    def is_argmax_invariant(self) -> bool:
+        """By censoring stop tokens, min-tokens can change the outcome
+        of the argmax operation in greedy sampling."""
+        return self._is_argmax_invariant
+
```

Comment:
The contents of this class looks like the min tokens LP ... is this just a copy-pasted temporary state before updating it with the wrapper LP impl?